datadog_checks_dev/setup.py

```diff
@@ -90,12 +91,15 @@
             'pyperclip>=1.7.0',
             'pysmi>=0.3.4',
             'semver',
-            'setuptools>=38.6.0',
             'tabulate>=0.8.9',
             'toml>=0.9.4, <1.0.0',
+            'tomli>=1.1.0',
+            'tomli-w>=1.0.0',
             'tox>=3.12.1',
             'twine>=1.11.0',
             'virtualenv>=20.5.0',
+            # TODO: Remove once every check has a pyproject.toml
```

Comment:
At what point will `datadog_checks_dev` be ported over?

Reply:
After this PR